doc/newsboat.asciidoc

```diff
@@ -129,21 +130,14 @@ documentation].
 include::chapter-firststeps.asciidoc[]
 
 
+=== Configuration Commands
 
-.Configuration Commands
-[frame="all", grid="all", format="dsv", separator="||", options="header", cols="10,15,15l,40,20l"]
-|======================================================================================================
-Configuration Command||Argument(s)||Default||Description||Example
-include::configcommands-linked.dsv[]
```

Comment:
Now that we don't use configcommands-linked.dsv, please remove all other mentions of this file from the repo (`git grep` is your friend ;)